CollAction/Services/Image/AmazonS3ImageService.cs

```diff
@@ -118,6 +129,22 @@ private byte[] ConvertImageToPng(Image<Rgba32> image)
             }
         }
 
+        private double GetScaleRatioForImage(Image<Rgba32> image)
+        {
+            if (image.Width > _imageResizeThreshold || image.Height > _imageResizeThreshold){
```

Comment:
The above line still has wrong braces formatting (must be on a newline)

Reply:
Maybe we start adding stylecop, then the compiler will start warning for badly formatted lines..

Reply:
Sory, fixed in 7a1eade, will use Rider in future.